lite/tests/classification/test_accuracy.py

```diff
@@ -197,61 +211,33 @@ def test_accuracy_with_example(
         assert m in actual_metrics
 
 
-def test_accuracy_with_image_example(
+def test_accuracy_with_image_example(  # TODO add comment about the deleted data here
     classifications_image_example: list[Classification],
 ):
     loader = DataLoader()
     loader.add_data(classifications_image_example)
     evaluator = loader.finalize()
 
     assert evaluator.metadata == {
-        "n_datums": 3,
-        "n_groundtruths": 4,
-        "n_predictions": 6,
-        "n_labels": 8,
-        "ignored_prediction_labels": [
-            ("k4", "v1"),
-            ("k4", "v8"),
-            ("k5", "v1"),
-            ("k4", "v5"),
-            ("k3", "v1"),
-        ],
-        "missing_prediction_labels": [
-            ("k5", "v5"),
-            ("k3", "v3"),
-        ],
+        "n_datums": 2,
+        "n_groundtruths": 2,
+        "n_predictions": 4,
+        "n_labels": 4,
+        "ignored_prediction_labels": ["v1", "v8", "v5"],
+        "missing_prediction_labels": [],
     }
 
     metrics = evaluator.evaluate(as_dict=True)
 
-    # test Accuracy
     actual_metrics = [m for m in metrics[MetricType.Accuracy]]
     expected_metrics = [
         {
             "type": "Accuracy",
-            "value": [0.3333333333333333],
-            "parameters": {
-                "score_thresholds": [0.0],
-                "hardmax": True,
-                "label": {"key": "k4", "value": "v4"},
-            },
-        },
-        {
-            "type": "Accuracy",
-            "value": [0.0],
-            "parameters": {
-                "score_thresholds": [0.0],
-                "hardmax": True,
-                "label": {"key": "k5", "value": "v5"},
-            },
-        },
-        {
-            "type": "Accuracy",
-            "value": [0.0],
+            "value": [0.5],  # TODO add comment about the bug here
```

Comment:
this value used to be .333, which (I believe) was a bug: if you look at `test_evaluate_image_clf` in `integration_tests/...`, this value should have been .5. no action needed, just explaining why this changed

<img width="701" alt="image" src="https://github.com/user-attachments/assets/7ad1e9c3-1901-465b-b162-5481d8d053f0">

Reply:
Great catch! The 1/3 accuracy is b/c we are dividing by `n_datums` in the computation. This resulted in a flawed output since that label key didnt exist for one datum.